src/independentTransforms.js

```diff
@@ -40,7 +40,14 @@
     applyHook: {
       callback: function (values) {
         var transformString = internalScope.toTransform(values);
-        return {transform: transformString + ' ' + values.transform};
+        return {
+          transform: transformString + ' ' + values.transform,
+          scaleForTesting: values.scale,
+          rotateForTesting: values.rotate,
+          translateForTesting: values.translate,
+          scale: '1 1 1',
+          rotate: '0deg',
+          translate: '0px'};
```

Comment:
File bug against Chrome to support none for these properties.
Add TODO to make these none and link to the bug.